apps/docs/content/components/forms/HelperMessage.mdx

```diff
@@ -1,48 +1,55 @@
 ---
-title: HelperMessage (WIP)
-description: TBD
+title: HelperMessage
+description: The HelperMessage component is used to display auxiliary text to guide users in the interface.
 category: "forms"
 links:
     source: https://github.com/gsoft-inc/wl-hopper/blob/main/packages/components/src/helpMessage/src/HelpMessage.tsx
 ---
 
-## Props
+<Example src="HelperMessage/docs/preview" isOpen />
 
-TODO: Add the props table, if we have multiple props table (ex: TagList and Tag) in the same file,  use the name of the component in ### before the each props table
+## Props
 
-## Guidelines
+<PropTable component="HelperMessage" />
 
-TODO: If we have some guidelines about this component's usage
+<FeatureFlag flag="next">
+    ## Guidelines
 
-### Accessibility ?
+    TODO: If we have some guidelines about this components usage
```

Comment:
it should have the '

Reply:
It breaks the build did not really pushed my fix as it does not breaks if it's not in the featureFlag component